base/src/main/java/com/smartdevicelink/proxy/rpc/VideoStreamingCapability.java

```diff
@@ -49,6 +49,10 @@
     public static final String KEY_DIAGONAL_SCREEN_SIZE = "diagonalScreenSize";
     public static final String KEY_PIXEL_PER_INCH = "pixelPerInch";
     public static final String KEY_SCALE = "scale";
+    /**
+     * @since SmartDeviceLink 7.0
```

Comment:
@kostyaBoss on line 53 you can change `7.0` to `7.0.0` (from the code generator tool).